planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/configschema/SchemaConfig.java

```diff
@@ -0,0 +1,65 @@
+package com.onthegomap.planetiler.custommap.configschema;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * An object representation of a vector tile server schema. This object is mapped to a schema YML file using SnakeYAML.
+ */
+public class SchemaConfig {
```

Comment:
Can we make this a record?  From what I can see the main issue is that `attribution` has a default value. If there's not a built-in jackson way to set a default value with annotations, we could work around that in the constructor or the getter, something like:

```java
@Override
public String attribution() {
  return attribution == null ? "default value" : attribution;
}
```

Reply:
Fixed in 4ffd340